libcxxabi/include/cxxabi.h

```diff
@@ -77,6 +77,9 @@ extern _LIBCXXABI_FUNC_VIS void __cxa_end_cleanup();
 #endif
 extern _LIBCXXABI_FUNC_VIS std::type_info *__cxa_current_exception_type();
 
+// GNU extension
```

Comment:
Can you please add some documentation explaining what this does and an example of when it can be called?